bcda/bcdacli/cli_test.go

```diff
@@ -812,8 +818,11 @@ func (s *CLITestSuite) TestBlacklistACO() {
 	s.Error(s.testApp.Run([]string{"bcda", "unblacklist-aco", "--cms-id", notFoundCMSID}))
 	s.Error(s.testApp.Run([]string{"bcda", "blacklist-aco", "--cms-id", notFoundCMSID}))
 
-	s.False(postgrestest.GetACOByUUID(s.T(), s.db, blacklistedACO.UUID).Blacklisted)
-	s.True(postgrestest.GetACOByUUID(s.T(), s.db, notBlacklistedACO.UUID).Blacklisted)
+	dbBlacklistedACO := postgrestest.GetACOByUUID(s.T(), s.db, blacklistedACO.UUID)
```

Comment:
Now that we have a name for it, should this be something like `newlyUnblacklistedACO`/`newlyBlacklistedACO`?

Reply:
Fixed